File: app/src/main/java/com/vkpapps/thunder/model/PhotoInfo.java
package com.vkpapps.thunder.model;
/***
 * @author VIJAY PATIDAR
 */
public class PhotoInfo {
    private String name,path;
    private boolean selected;
    public PhotoInfo(String name, String path) {
        this.name = name;
        this.path = path;
    }

    public String getName() {
        return name;
    }

    public boolean isSelected() {
        return selected;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }
}

File: app/src/main/java/com/vkpapps/thunder/model/PhotoInfo.kt
package com.vkpapps.thunder.model

import androidx.annotation.NonNull
import androidx.room.Entity
import androidx.room.PrimaryKey

/***
 * @author VIJAY PATIDAR
 */
@Entity
class PhotoInfo(var name: String, var path: String) {
    @PrimaryKey
    @NonNull
    lateinit var id: String
    var isSelected = false
    var modified: Long = 0
}